docs/pages-for-subheaders/rancher-v2.6-hardening-guides.md

```diff
@@ -31,26 +31,23 @@ These guides have been tested along with the Rancher v2.6 release. Each self-ass
 
 | Kubernetes Version | CIS Benchmark Version | Self Assessment Guide | Hardening Guides |
 | ------------------ | --------------------- | --------------------- | ---------------- |
-| Kubernetes v1.18 up to v1.23 | CIS v1.6 | [Link](../reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-self-assessment-guide-with-cis-v1.6-benchmark.md) | [Link](../reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-hardening-guide-with-cis-v1.6-benchmark.md) |
+| Kubernetes v1.18 | CIS v1.6 | [Link](../reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-self-assessment-guide-with-cis-v1.6-benchmark.md) | [Link](../reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-hardening-guide-with-cis-v1.6-benchmark.md) |
+| Kubernetes v1.19 up to v1.21 | CIS v1.20 | [Link](../reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-self-assessment-guide-with-cis-v1.20-benchmark.md) | [Link](../reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-hardening-guide-with-cis-v1.20-benchmark.md) |
+| Kubernetes v1.22 up to v1.24 | CIS v1.23 | [Link](../reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-self-assessment-guide-with-cis-v1.23-benchmark.md) | [Link](../reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-hardening-guide-with-cis-v1.23-benchmark.md) |
```

Comment:
With the dev of CIS-1.24 (k8s v1.24) and CIS-1.7 (k8s v1.25), I suggest to update L36 to have CIS-1.23 covering up to k8s v1.23:

```
- Kubernetes v1.22 up to v1.24
+ Kubernetes v1.22 up to v1.23
```